### PR TITLE
Changed paths to lead to btgsuit, changed header size to work with BTG

### PR DESCRIPTION
**Pull request by Vutov**
go.mod was changed by hand. Once we have this in BTCGPU we should update the btgwallet. Once its committed we should come back to the BTCGPU/neutrino and fix the mod.go. Or at least I think it should be in this order.

### CODE REVIEW

File: headerlist/header_list.go [hunk 1]
```diff
@@ -44,3 +44,61 @@ type Node struct {
...
+
+// Equals compares two Nodes if they have the same properties.
+// A structure is comparable only if it contains no slice/array.
+// wire.BlockHeader has Solution byte[]
+func (n *Node) Equals(o Node) bool {
+	if n == nil {
+		return false
+	}
+
+	if n.Height != o.Height {
```
Comment (h4x3rotab) [inline headerlist/header_list.go:56]: Not necessary as Equials(h1, h2) covers all these cases.

Comment (Vutov) [reply headerlist/header_list.go:56]: Not exactly. Equals covers the Height inside of the BlockHeader object, there is Height on Block level as well. On theory its possible Height inside and Height outside to be different or one of them to not be populeted,

Comment (h4x3rotab) [reply headerlist/header_list.go:56]: Make sense

**Review by h4x3rotab — APPROVED** [review]
LGTM

**Pull request merged**